Makefile

```diff
@@ -209,6 +211,11 @@ build: build-go build-js ## Build backend and frontend.
 run: $(BRA) ## Build and run web server on filesystem changes.
 	$(BRA) run
 
+.PHONY: run-go
+run-go: ## Build and run web server immediately.
+	$(GO) run -race $(if $(GO_BUILD_TAGS),-build-tags=$(GO_BUILD_TAGS)) \
```

Comment:
I think the we have to remove `-race` from here otherwise it will always be enabled

Reply:
Even with this flag removed it seems it's still always enabled even if you don't have a `.go-race-enabled-locally` file

Reply:
Yes! `make run` and `make run-go` are forced to use `-race` now, since they are only used locally. This is not blocking running the process, it will just dump the race and continue with your work. The opt-in part is there only to prevent using `-race` in Drone and pipelines, so we require manually enabling the race detector, but we ideally want to start using `-race` everywhere and all the time. It's just like another `go vet`/linter/etc.
We can't at this point enable it globally because that would break all pipelines all the time and would make it too painful. So we make it opt-in part allows you to also find data races for locally built binaries. I'll make a followup PR to try to add `-race` to local tests as well.

Reply:
Is there a way to opt-out of the race detection? On my local environment there is a very significant performance impact and I'm not sure I can do much about the race conditions when I'm just working on the front-end :)

Reply:
@gillesdemey the performance impact should be noticeable only the first few times you compile and run everything. The Race Detector works by adding instrumentation code to every function, which means it needs to recompile everything from scratch the first time (even the standard library), so you don't have the speed of the already cached compiled objects. A few runs later the performance impact should be negligible again due to caching.

> I'm not sure I can do much about the race conditions when I'm just working on the front-end

Yes you can! Please, consider pinging someone on Slack. Data Race Detection is opportunistic, meaning that we need many runs to detect them. This means that we also rely on all devs as a means to provide herd immunity against this kind of bugs, which can have very nasty effects like crashes and (worst of all) silent data corruption. Don't mind being noisy on Slack, we should be fixing these bugs to make the experience better for everyone.

> Is there a way to opt-out of the race detection?

Yes! You can delete the file `.go-race-enabled-locally`. If you decide to do so due to:

1. The performance impact: please, consider the above explanation. If the impact is consistently noticeable (say, >30 seconds compared to the previous state?), we might need to investigate it, please flag it. We would really appreciate everyone's help, which means we also need to care for the impact. So please, feel free to ping me on Slack if you're experiencing unusual performance drains after a few runs.
2. The verbosity of errors: it is understandable that you have other priorities right now, reporting issues takes time (and context switching), and as this is the first time we enable the Race Detector we know there are some dangling bugs that will pop up. However, we expect to have them fixed soon and seeing them should be a rare thing later.

If you are still not convinced, please, consider re-enabling it later in a few weeks. If there had been an abnormal performance impact we may have solved it by then (but remember the first few runs will still be slower), and the verbosity will be significantly reduced due to fixing the current dangling items we need to fix now.